snippets/mcs-endpoint-limitation.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/installation-about-custom-azure-vnet.adoc
+// * modules/machine-config-operator.adoc
+// * security/certificate_types_descriptions/machine-config-operator-certificates.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+There is no supported way to block or restrict the Machine Config Server (MCS) endpoint in the current architecture. The MCS must be exposed to the network so that newly-provisioned machines, which have no existing configuration or state, are able to fetch their configuration. In this model, the root of trust is the certificate signing requests (CSR) endpoint, which is where the kubelet sends its certificate signing request for approval to join the cluster. Because of this, machine configs should not be used to distribute sensitive information.
```

Comment:
@mburke5678 I think we should explicitly call out secrets:
> Because of this, machine configs should not be used to distribute sensitive information, **such as secrets**.